src/Frontend/Content/Assets.php

```diff
@@ -58,8 +59,9 @@ public function __invoke(Document $document, Request $request)
 
     private function commit(array $compilers)
```

Comment:
Can we add a comment saying "This method only runs when in debug mode, so we force a recompilation"?